test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -46,20 +55,42 @@ public static String getConfig(final String key, final String defaultValue) {
     }
 
     public static Service createZookeeperService() {
-        return DOCKER_BASED ? new ZookeeperDockerService("zookeeper")
-                : new ZookeeperService("zookeeper");
+        switch (EXECUTOR_TYPE) {
+            case REMOTE_SEQUENTIAL:
+                return new ZookeeperService("zookeeper");
+            case DOCKER:
+                return new ZookeeperDockerService("zookeeper");
+            case K8s:
```

Comment:
I would always use the full name instead of the abbreviation.

Reply:
done.